examples/meteor/v1beta1/meteor-controller.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Ah, yes,. I thought the guidance was to copy the old files into a subdir for now. If they're not needed, I'll remove them.